doc/manuals/admin/database_model.md

```diff
@@ -288,6 +291,13 @@ Fields:
     (meaning JSON notifications in NGSIv1 format), **normalized**, **keyValues** and **values** (the last three used in NGSIv2 format).
 -   **status**: either `active` (for active subscriptions) or `inactive (for inactive subscriptions).
 -   **description** (optional field): a free text string describing the subscription. Maximum length is 1024.
+-   **custom**: a boolean field to specify if this subscription uses customized notifications (a functionality in the NGSIv2 API).
```

Comment:
This could be not aligned with the current implementation (as probably we are using "extended" instead of "custom") but I have preferred to include in this PR the "final snapshot" by the end of sprint.